src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1108,6 +1109,11 @@ void ChordLayout::layoutArticulations3(Chord* item, Slur* slur, LayoutContext& c
 //! May be called again when the chord is added to or removed from a beam.
 void ChordLayout::layoutStem(Chord* item, LayoutContext& ctx)
 {
+    if ((item->staffType() && item->staffType()->xmlName() == "stdJianpu")
+        && (item->staffType() && item->staffType()->lines() == 0)) {
+        return;
+    }
```

Comment:
Instead of this addition, it would be better to check the staff type in `shouldHaveHook`/`shouldHaveStem`, so that the hook/stem will be properly removed when changing the staff type from normal to Jianpu.